search-sentiments-map/routes/search.js

```diff
@@ -204,16 +214,18 @@ function getSentiment(searchResult) {
   });
 }
 
-/** Deletes search results from 7 days ago. */
+/** Deletes stale search results. Currently deletes after 7 days. */
```

Comment:
Since you're including the duration in the threshold, you can leave out the part that talks about the time.

This will also avoid potentially missing out on updating the comment if you change the threshold later.